core/services/ocr2/delegate.go

```diff
@@ -1201,9 +1231,36 @@ func (d *Delegate) newServicesOCR2Keepers20(
 		return nil, fmt.Errorf("keepers2.0 services: failed to get chain (%s): %w", rid.ChainID, err2)
 	}
 
-	keeperProvider, rgstry, encoder, logProvider, err2 := ocr2keeper.EVMDependencies20(jb, d.db, lggr, chain, d.ethKs)
-	if err2 != nil {
-		return nil, errors.Wrap(err2, "could not build dependencies for ocr2 keepers")
+	transmitterID := spec.TransmitterID.String
+	relayer, err := d.RelayGetter.Get(rid)
+	if err != nil {
+		return nil, ErrRelayNotEnabled{Err: err, Relay: spec.Relay, PluginName: "mercury"}
+	}
+
+	provider, err := relayer.NewPluginProvider(ctx,
```

Comment:
I would say let's leave 2.0 code untouched. we want to deprecate it in Q1, and changing this adds risk + testing load

Reply:
@jmank88 would you have any objections to leaving the 2.0 code as-is, non loopp'd? @infiloop2 I guess if we were to keep 2.0 untouched, we would want to keep 2.0 using the pipeline transmitter, is that okay with you?

Reply:
I think that is OK

Reply:
@infiloop2 I should say that we can, and that it's easier, to have 2.0 use the default contract transmitter, since that's already been tested and merged, but want to be sure before continuing 👍

Reply:
ah if that's already been merged then let's leave it as is. Overall we plan to completely deprecate 2.0 in Q1 and nuke this code all together